shell/app/i18n-easy-edit/overwrite_i18n.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import i18n from 'core/i18n';
+import I18nWrapper from './pages/i18n-wrapper';
+import { getTransFromLocalStorage, splitKey } from './utils';
+
+// overwrite i18n.t
+const originT = i18n.t;
```

Comment:
Suggestion：the params for this function should be i18n itself,  then you dont need to import `import i18n from 'core/i18n'`, this behavior makes this func decoupe with `core/i18n`

Reply:
i18next is not exit in shell